lib/zones/route_data.js

```diff
@@ -17,6 +17,7 @@ module.exports = function(data){
 				oldData = can.route.data;
 				intercept(can.route, "data", function(data){
 					viewModel = data;
+					viewModel.bind("statusCode", noop);
```

Comment:
What does `intercept` do?  Run on a set?  Does this work with IE8? Just want to make sure this code doesn't run everytime the zone switches.